sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -37,6 +44,50 @@ export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefine
   return timeoutInMs;
 }
 
+/**
+ * The set of options to configure request cancellation.
+ * - `abortSignal` : A signal used to cancel an asynchronous operation.
+ */
+export interface AbortSignalOptions {
+  /**
+   * An implementation of the `AbortSignalLike` interface to signal the request to cancel the operation.
+   * For example, use the &commat;azure/abort-controller to create an `AbortSignal`.
```

Comment:
is this encoded &commat; intentional?

Reply:
Yes for now. This is currently necessary or else typedoc won't properly render the `@` symbol.